schemas/2016-05-01-preview/Microsoft.MachineLearning.json

```diff
@@ -383,9 +379,7 @@
           "oneOf": [
             {
               "type": "object",
-              "additionalProperties": {
-                "type": "object"
-              }
```

Comment:
So we are not loosing any information here ? Is this still going to be seen as a dictionary ?